web_programming/get_google_search_results.py

```diff
@@ -1,36 +1,96 @@
+import urllib.parse
+
 import requests
-from fake_useragent import UserAgent
+from bs4 import BeautifulSoup
+from requests_html import HTMLSession
 
+BASE_URL = "https://www.google.com"
 
-def get_google_search_results(query: str = "potato") -> str:
-    """Return a Google search result page for the given query.
 
+def parse_results(query: str = "") -> list:
+    """ "Reads google search result page for the given query and
+    stores all the results in with title, link and descriptions
+    in a list.
     Args:
-        query: The search term provided by the user. Defaults to "potato".
-
-    >>> "hacktober" in get_google_search_results("hacktober")
-    True
-    >>> "potato" in get_google_search_results()
-    True
```

Comment:
Why remove all tests?

Reply:
wait, I think I mistakenly removed this while merging. This is my first time trying to contribute to a public repo like this and I am really sorry for all these continuous mistakes. I will fix this. I appreciate your patience, thanks.